expressEngine/model/lib/index.js

```diff
@@ -1,9 +1,9 @@
 class ModelLib {
   constructor() {
-    this.roleEnum = ['system-admin', 'supplier'];
-  };
+    this.roleEnum = ["system-admin", "supplier"];
```

Comment:
@aman-successive Everywhere you are using single quote, what is the use of double quotes now.

Reply:
changed it